src/services/file/upload-file-to-presigned-url.ts

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+
+import api from "@/lib/axios";
+import { ENDPOINT } from "@/services/endpoint";
+import { FilePresignedUrlResponse } from "@/types/file";
+
+interface GetPresignedUrlParams {
+  prefix: string;
+  fileName: string;
+}
+
+export const getPresignedUrl = async ({
+  prefix,
+  fileName,
+}: GetPresignedUrlParams) => {
+  const { data } = await api.get<FilePresignedUrlResponse>(
+    ENDPOINT.FILE.GET(prefix, fileName),
+  );
+  return data.presignedUrl;
+};
+
+interface UploadFileToPresignedUrl {
+  presignedUrl: string;
+  file: File;
+}
+
+export const putFileToPresignedUrl = async ({
+  presignedUrl,
+  file,
+}: UploadFileToPresignedUrl) => {
```

Comment:
이 타입 이름 위쪽 `getPresignedUrl`, `GetPresignedUrlParams`와 동일한 패턴으로 수정하는 게 어떨까요? 일관성이 있으면 좋을 것 같습니다.

```suggestion
interface PutFileToPresignedUrlParams{
  presignedUrl: string;
  file: File;
}

export const putFileToPresignedUrl = async ({
  presignedUrl,
  file,
}: PutFileToPresignedUrlParams) => {
```